guardian/actions-merge-release-changes-to-protected-branch
# Run build step in pre-commit hook

## What does this change?

This PR adds husky and configures a pre-commit hook to generate the build and add the contents of the `dist` directory into the commit.

## How to test

Make a change that would produce a change in the build output. Commit the change and see the build has been updated and included in the commit.

## How can we measure success?

The build is generated and committed auto-magically. 

## Have we considered potential risks?

The hook commits something in the `dist` directory that the user wasn't expecting
